hera_sim/beams.py

```diff
@@ -249,3 +249,741 @@ def interp(self, az_array, za_array, freq_array, reuse_spline=None):
             interp_data += (1.0 - step) * (mainlobe_pert - mainlobe0)
 
         return interp_data, interp_basis_vector
+
+
+class ZernikeBeam(AnalyticBeam):
```

Comment:
Any reason this doesn't subclass the `PolyBeam`?

Reply:
Yes, it implements a 2D polynomial basis instead of 1D + various transformations (rotation/stretch/etc), so there's not really anything to be gained from subclassing.